proto/query.proto

```diff
@@ -262,6 +262,7 @@ message ExecuteOptions {
     UNSPECIFIED = 0;
     OLTP = 1;
     OLAP = 2;
+    MIGRATION = 3;
```

Comment:
Will do. That name fits better in the vitess nomenclature.  I'll also fix the test.